pyglider/seaexplorer.py

```diff
@@ -454,4 +454,3 @@ def _parse_sensor_config(filename):
     return active_device_dicts
 
 
-__all__ = ['raw_to_rawnc', 'merge_rawnc', 'raw_to_timeseries']
```

Comment:
revert this

Reply:
Done.

Reply:
OK, but please add the extra line back in and remove the two spare carriage returns.  We don't have linting turned on, but we will ;-). Liniting ensures some coding consistency and it will complain about both of these changes.  

basically a little trick here is `git restore --source=upstream/main pyglider/seaexplorer.py` to restore the old version.  

Please do the same for slocum below.

Reply:
Done.